fractal-puppy/Engine
# Create a new compression type

to test:

- selected a resource texture on resource manager.
- right click button on mouse
- selected IMPORT CONFIGURATION
- select NO_COMPRESSION on dropdown list
- accept
- restart the engine
- load a scene
- selected a gameobject
- in configuration, check show (material)
- in diffuse, selected the changed texture.
